lib/pages/playlists_page.dart

```diff
@@ -80,6 +81,14 @@ class _PlaylistsPageState extends State<PlaylistsPage> {
                           onPressedIcon: () => context
                               .read<PlaylistStore>()
                               .removePlaylistById(item.id),
+                          onTap: () => Navigator.of(context).push(
+                            MaterialPageRoute<PlaylistItemPage>(
```

Comment:
I would define this route in `PlaylistItemPage`, also, `PlaylistItemPage` is not the actual generic parameter here, it's `void`